etc/onionjuggler/conf.d/30_whonix.conf

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env sh
+## Configuration file for OnionJuggler
+##
+## DO NOT EDIT THIS FILE!!
+##
+## This file defines the default service selection as shipped in a
+## release. Upgrades of OnionJuggler will modify this file.
+##
+## To select the service options you desire, please override these
+## options in the file /etc/onionjuggler/conf.d/*.conf
```

Comment:
Maybe change this to:
```
## Please create and add modifications to the following file instead:
## /etc/onionjuggler/conf.d/50_user.conf
```
The problem is that it could be forgotten on updates to modify this header comment.